packages/edit-site/src/components/dataviews/view-table.js

```diff
@@ -472,6 +548,10 @@ function ViewTable( {
 												header.column.columnDef
 													.maxWidth || undefined,
 										} }
+										className={
+											header.column.columnDef.className ||
```

Comment:
What is this for? I don't see any `className` prop in the pages fields' definition.